include/openthread/platform/radio.h

```diff
@@ -443,19 +454,17 @@ void otPlatRadioSetPromiscuous(otInstance *aInstance, bool aEnable);
  * @param[in]   aInstance    A pointer to an OpenThread instance.
  * @param[in]   aKeyIdMode   The key ID mode.
  * @param[in]   aKeyId       Current MAC key index.
- * @param[in]   aKeySize     The key size.
  * @param[in]   aPrevKey     A pointer to the previous MAC key.
  * @param[in]   aCurrKey     A pointer to the current MAC key.
  * @param[in]   aNextKey     A pointer to the next MAC key.
  *
  */
-void otPlatRadioSetMacKey(otInstance *   aInstance,
-                          uint8_t        aKeyIdMode,
-                          uint8_t        aKeyId,
-                          uint8_t        aKeySize,
-                          const uint8_t *aPrevKey,
-                          const uint8_t *aCurrKey,
-                          const uint8_t *aNextKey);
+void otPlatRadioSetMacKey(otInstance *    aInstance,
```

Comment:
```suggestion
void otPlatRadioSetMacKey(otInstance     *aInstance,
```

Reply:
Same here. This is enforced by clang-format and part of our style. The pointer `*` is aligned with the type not the variable name.